### DISCUSSION TITLE
I have a nested route. Is there any way to avoid using the parent route layout for a page?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#19105] [Questions (Nuxt 2)]
I have a nested route. Is there any way to avoid using the parent route layout for a page?
Login.vue has a way not to use test Vue layout?

/test.vue
/test
/test/index.vue
/test/login.vue

**Comment by Pascazio** [comment]
Hi! in the nuxt.js documentation you can find this page : "https://nuxtjs.org/docs/directory-structure/layouts".
 
Here you can find how to correctly set the layouts folder. However, in a nutshell: you should create into your "layouts" folder all of your templates.vue. In this way, you can add into every single page, in the script section -> layout: 'layoutNameExample'

This page will use layouts/layoutNameExample.vue layout.

Hope this would help!

**Comment by llanting** [comment]
I had the same issue and found [this older issue](https://github.com/nuxt/nuxt.js/issues/1425) with a solution that worked for me:

`created () {
  this.$root.setLayout('other')
}`

**Comment by bbhxwl** [comment]
> I had the same issue and found [this older issue](https://github.com/nuxt/nuxt.js/issues/1425) with a solution that worked for me:
> 
> `created () { this.$root.setLayout('other') }`

I mean to cancel the nested route, not the layout.
